packages/rum-core/src/common/url.js

```diff
@@ -137,21 +151,23 @@ class Url {
     this.protocol = protocol || location.protocol
 
     /**
-     * Construct port and hostname from host and assign defaults based
-     * on http and https protocols
+     * Construct port and hostname from host
+     *
+     * Port numbers are not added for default ports of a given protocol
+     * and hostname would match host when port is not present
      */
+    this.hostname = this.host
+    this.port = ''
     if (/:\d+$/.test(this.host)) {
```

Comment:
Wouldn't it be simpler to do what is done here: https://github.com/unshiftio/url-parse/blob/master/index.js#L41, i.e. adding a new rule for the port?

Reply:
It's actually slower to do that way, since the for loop has to check for each instruction whether it is a Regex thats why we don't use that.

Reply:
I don't think the perf impact is actually significant since the loop goes over a constant array of only 6 items. I'm ok with keeping it this way though.